docs/guidance/environment-variables.md

```diff
@@ -0,0 +1,72 @@
+# Network-related environment variables 
+
+## CLUSTER_DOMAIN
+- **Default**: `cluster.local`
+- **Configurable**: `true`
+- **Location**: KubeRay operator
+- **Description**: The cluster domain name of the Kubernetes cluster in which the KubeRay operator is installed.   
+It is for facilitating communication and DNS-based service discovery within a Kubernetes cluster.   
+It forms part of Fully Qualified Domain Names (FQDNs) used for addressing services and resources.  
+See [kuberay/pull/951](https://github.com/ray-project/kuberay/pull/951).
+
+## FQ_RAY_IP
+- **Default**: `${HEAD_SERVICE}.${NAMESPACE}.svc.${CLUSTER_DOMAIN}`
+- **Configurable**: `false`
+- **Location**: Worker pods only
+- **Description**: Fully qualified domain name(FQDNs) of head service.  
+It is used to access head service within a Kubernetes cluster. This variable is set only in worker pods, while users can utilize 'localhost' when inside the head pod. 
+See [kuberay/pull/938](https://github.com/ray-project/kuberay/pull/938).
+
+## RAY_IP
+- **Default**: `${HEAD_SERVICE}`
+- **Configurable**: `false`
+- **Location**: Worker pods only
+- **Description**: Name of head service.   
+It is used to access head service within **the same namespace** in a Kubernetes cluster. This variable is deprecated and should be maintained solely for backward compatibility purposes. Instead, users are advised to use FQ_RAY_IP for this purpose.
```

Comment:
This variable is intended for worker Pods to access the head Pod, but it is now deprecated. It should only be maintained for backward compatibility purposes. Instead, we recommend using `FQ_RAY_IP` for this task. See [kuberay/#938](https://github.com/ray-project/kuberay/pull/938) for more details.